pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -38,7 +38,7 @@ For the very best experience with all Code.org content, we recommend consulting
 | **To use YouTube hosted videos** <br/>(Deprecated late July 2018)| `https://s.youtube.com/*`<br/>`https://www.youtube.com/*`<br/>`https://*.googlevideo.com/*`<br/>`https://*.ytimg.com/*`                                |
 | **To use Code.org hosted videos**                                                  | **Unblock:**<br/>`https://videos.code.org`<br/>**Block:**<br/>`https://www.youtube.com/favicon.ico`<br/>`https://www.youtube-nocookie.com/favicon.ico` |
 | **To use Internet Simulator**   | `https://api.pusherapp.com`<br/>`wss://ws.pusherapp.com:443`                                                                                           |
-| **To use Web Lab** | `https://downloads.computinginthecore.org`<br/>`https://codeprojects.org`                                                                              |
+| **To use Web Lab** | `https://codeprojects.org`<br/>`https://*.codeprojects.org`                                                                              |
```

Comment:
This is a change to the "Sites to unblock" section of https://code.org/educate/it.  After this change, we'll no longer need to list downloads.computinginthecore.org as a requirement for Web Lab.  I've listed both `codeprojects.org` and `*.codeprojects.org` separately here because we do the same thing for `code.org` and `*.code.org` above.